pinot-tools/src/main/java/org/apache/pinot/tools/admin/command/DeleteTableCommand.java

```diff
@@ -97,6 +72,7 @@ public String toString() {
     String retString =
         ("DeleteTable -tableName " + _tableName + " -type " + _type + " -retention " + _retention
             + " -controllerProtocol " + _controllerProtocol + " -controllerHost " + _controllerHost
+            + Arrays.toString(_headers)
```

Comment:
Missing key "-headers". Consider always putting headers in the end before "-exec" for consistency